### DISCUSSION TITLE
what version of pytorch or rastervision

### DISCUSSION THREAD

**Discussion by georgeper2001** [azavea/raster-vision#2104] [General]
in my script i have the following:        "model = ObjectDetectionModelConfig(external_def=ExternalModuleConfig("
and the following: "backend = PyTorchObjectDetectionConfig(
        model=model,"
i get the error: "pydantic.error_wrappers.ValidationError: 1 validation error for PyTorchObjectDetectionConfig
model
  external_def is currently not supported for Object Detection. (type=value_error.config)"

was wondering what version of rastervision or pytorch has external_def as a parameter for model for PyTorchObjectDetectionConfig?

**Comment by AdeelH** [comment]
`PyTorchObjectDetectionConfig` (and `external_def`) is a Raster Vision thing and not a PyTorch thing. Support for `external_def` for object detection was added in v0.20. You must be using an older version than that. I recommend updating to the latest version, v0.21.
